### PR TITLE
Improve Source Reconcile loop

### PR DESCRIPTION
**Pull request by zroubalik**
Signed-off-by: Zbynek Roubalik <zroubali@redhat.com>

Minor fixes and chores in log messages and events

Fixes: #61

### CODE REVIEW

File: pkg/reconciler/source/source.go [hunk 1]
```diff
@@ -59,25 +59,22 @@ func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
...
-		logging.FromContext(ctx).Errorw("Source in work queue no longer exists")
+		logging.FromContext(ctx).Infow("Source in work queue no longer exists")
 		return nil
 	} else if err != nil {
+		logging.FromContext(ctx).Errorw("not able to get runtime object")
 		return err
 	}
 
-	logging.FromContext(ctx).Info("RECONCILE kind: " + runtimeObj.GetObjectKind().GroupVersionKind().String())
+	logging.FromContext(ctx).Infow("Reconcile Knative Source", "kind", runtimeObj.GetObjectKind().GroupVersionKind().String())
```
Comment (lionelvillard) [inline pkg/reconciler/source/source.go:77]: Do you want to use zap.String here?

Comment (zroubalik) [reply pkg/reconciler/source/source.go:77]: What is the benefit?

Comment (lionelvillard) [reply pkg/reconciler/source/source.go:77]: Just to be consistent with the log line above. You decide, I'm fine with both.

Comment (zroubalik) [reply pkg/reconciler/source/source.go:77]: Makes sense!

Comment (zroubalik) [reply pkg/reconciler/source/source.go]: @lionelvillard fixed, PTAL

**Review by lionelvillard — APPROVED** [review]

**Pull request merged**